src/pages/dar_application/collaborator/CollaboratorSummary.js

```diff
@@ -0,0 +1,122 @@
+import { a, div, span } from 'react-hyperscript-helpers';
+
+export const CollaboratorSummary = (props) => {
+  const {
+    collaborator,
+    index
+  } = props;
+
+  const cardStyle = {
+    display: 'flex',
+    padding: '2% 5%',
+    border: '1px solid #0948B7',
+    borderRadius: '5px',
+    background: '#FFFFFFFF',
+    boxShadow: 'rgb(234, 227, 227) -4px 6px 9px 0px',
+    marginBottom: '2%',
+    margin: '1.5rem 0 1.5rem 0',
+    justifyContent: 'space-around',
+  };
+
+  const editButtonStyle = {
+    display: 'flex',
+    justifyContent: 'flex-end',
+    color: '#0948B7',
+    marginTop: 25,
+    padding: '0 1%',
+  };
+
+  const deleteButtonControlStyle = {
+    display: 'flex',
+    justifyContent: 'flex-end',
+    marginTop: 25,
+    padding: '0 1%',
+  };
+
+  return div({}, [
+    div({}, [
+    ]),
+    div({
+      id: index,
+
+      style: cardStyle,
+    }, [
+      div({
+        style: {
+          flex: '1 1 100%',
+          marginRight: '1.5rem',
+        }
```

Comment:
Should we be pulling all of this css into a css file similarly to what we're doing for other parts of [the new dar application](https://github.com/DataBiosphere/duos-ui/tree/develop/src/pages/dar_application)?